assets/src/scripts/monitoring-location/components/hydrograph/date-controls.spec.js

```diff
@@ -205,8 +205,8 @@ describe('monitoring-location/components/hydrograph/date-controls', () => {
     });
 
     it('Expects an alert to be thrown if the end date is earlier than the start date.', () => {
-        select('#custom-start-date').property('value', '2063-04-05');
-        select('#custom-end-date').property('value', '2063-04-03');
+        select('#custom-start-date').property('value', '04/05/2063');
+        select('#custom-end-date').property('value', '04/03/2063');
```

Comment:
These changes just reflect the date format change needed to work with the USWDS component. The store is still keeping the ISO format.